### PR TITLE
Replace BaseCommand.option_list with add_arguments() method

### PR DESCRIPTION
**Pull request by dongbohu**
This PR addresses issue #16.

Based on the discussion in  issue #17, the `--taxonomy_id` flag in `genes_load_wb.py` is also removed because it is not being used.

### CODE REVIEW

**Comment by dongbohu** [comment]
@dhimmel: This may seem like a big PR, but what the revisions do are pretty boring. It simply replaces `make_options` with `add_arguments()` method (similar to the PR you reviewed yesterday).

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,5 @@
 *~
 *.pyc
+dist/
+django_genes.egg-info/
+.doctrees/
```
Comment (dhimmel) [inline .gitignore]: missing final newline

File: README.rst [hunk 2]
```diff
@@ -147,14 +147,14 @@ sub-directory:
    * (Required) taxonomy_id: taxonomy ID for organism for which genes
      are being populated;
 
-   * (Optional) systematic_col: systematic column in gene info file.
-     Default is 3;
+   * (Optional) gi_tax_id: alternative taxonomy ID for some organisms
+     (such as S. cerevisiae);
```
Comment (dhimmel) [inline README.rst:151]: It's repetitive to have documentation in README and command line help message. Consider [what we do for Manubot](https://github.com/manubot/manubot/blob/81394c0a8153fa3ad41ee3cb987227c8ea043125/README.md#process) (potentially in another PR), to have the README usage documentation come directly from the utilities help message.

**Review by dhimmel — APPROVED** [review]

File: README.rst (hunk 2 above)
Comment (dongbohu) [reply README.rst:151]: This `REAME.rst` was actually generated by sphinx based on the template in `sphinx` directory.

File: .gitignore (hunk 1 above)
Comment (dongbohu) [reply .gitignore]: Thanks. Done.

**Pull request merged**